### DISCUSSION TITLE
How to read from JSON File in Maestro ?

### DISCUSSION THREAD

**Discussion by gr33nS34l** [mobile-dev-inc/Maestro#1942] [Q&A] [answered]
I want to implement a solution where you can access the native android string Ressources
I already wrote a script for creating an strings.json out of the strings.xml but can not find a good solution how to read from this files in the maestro test. 

Sadly after some research i found out that this seems not to be supported by maestro. 
Is there something planned in the future or will maestro not be able to offer such solutions ?

**Comment by billomore** [comment, marked as answer]
didn't try it, but it looks like you can spin up a simple express server to serve the JSONs and read them via HTTP https://docs.maestro.dev/advanced/javascript/make-http-s-requests
